src/components/PageLayout.styles.tsx

```diff
@@ -1,17 +1,18 @@
 import tw, { styled } from "twin.macro";
 
-export const PageBody = styled.div`
+export const PageCointainer = styled.div`
```

Comment:
I think this is supposed to be `PageContainer`

Reply:
Yeah, a typo 😄 But I actually like the wordplay - Cointainer = Wallet 😂